automated/linux/fuego-multinode/README.MD

```diff
@@ -0,0 +1,127 @@
+
+# Using Fuego with LAVA MultiNode job
+
+1. Build and publish standalone Fuego docker image.
+
+   Fuego project provides dockerfiles to build docker image locally, and the project has a plan to release official docker images. Before that happens, for the usage with LAVA, we need to build and publish test code included standalone Fuego docker image with the following patch applied.
+
+    ```
+    $ git clone https://bitbucket.org/fuegotest/fuego.git
+    $ cd fuego
+    $ git diff
+    diff --git a/Dockerfile b/Dockerfile
+    index 269e1f6..927199c 100644
+    --- a/Dockerfile
+    +++ b/Dockerfile
+    @@ -218,6 +218,23 @@ RUN ln -s /fuego-ro/scripts/fuego-lava-target-teardown /usr/local/bin
+    #RUN DEBIAN_FRONTEND=noninteractive apt-get update
+    #RUN DEBIAN_FRONTEND=noninteractive apt-get -yV install crossbuild-essential-armhf cpp-arm-linux-gnueabihf gcc-arm-linux-gnueabihf binutils-arm-linux-gnueabihf
+
+    +# ==============================================================================
+    +#Install fuego
+    +# ==============================================================================
+    +RUN git clone https://bitbucket.org/fuegotest/fuego.git /fuego \
+    +    && git clone https://bitbucket.org/fuegotest/fuego-core.git /fuego-core \
+    +    && ln -s /fuego/fuego-ro/ / \
+    +    && ln -s /fuego/fuego-rw/ /
+    +
+    +
+    +# ==============================================================================
+    +#Install arm64 and armhf toolchain
+    +# ==============================================================================
+    +RUN /fuego-ro/toolchains/install_cross_toolchain.sh arm64 \
+    +    && /fuego-ro/toolchains/install_cross_toolchain.sh armhf \
+    +    && apt-get clean \
+    +    && rm -rf /tmp/* /var/tmp/*
+    +
+    # ==============================================================================
+    # Setup startup command
+    ```
+
+   E.g. https://cloud.docker.com/repository/docker/chaseqi/standalone-fuego/tags
+
+2. Add/modify docker device to LAVA as Fuego host controller.
+
+   Docker device dictionary example:
+    ```
+    {% extends 'docker.jinja2' %}
+    {% set docker_cpus = '2' %}
+    {% set docker_memory = '2048M' %}
+    {% set docker_devices = ["/dev/kvm:/dev/kvm"] %}
+    {% set docker_volumes = ["/var/lib/lava/dispatcher/tmp:/var/lib/lava/dispatcher/tmp", "/boot:/boot", "/lib/modules:/lib/modules"] %}
+    ```
+    
+    Refer to https://staging.validation.linaro.org/scheduler/device/staging-docker-03/devicedict
+
+    Notes: 
+    * At the time of writing this guide, fuego master branch, [v1.4.0](https://bitbucket.org/fuegotest/fuego/commits/8b16eda803d4627f66a0102eac48d00b6857c801), still depends on Jenkins for test run from command line. Jenkins is memory hungry, 1G isn't enough, 2G is recommended.
```

Comment:
fwiw, it's inaccurate. Jenkins can run on 512M. Memory usage depends on the number of nodes connected and the number of concurrent build jobs, etc...

Reply:
That is right, adding new fuego nodes eats more. We started with 1G, but that is not enought, adding node hangs. It is recorded here https://projects.linaro.org/browse/LSS-531

Reply:
So I believe this bit can be rewritten with the context, something like: Our testing of Fuego container showed at least 2G are required, defined in the docker device dictionary using docker_memory.

Reply:
Thanks for advice. Updated.